test-suite/output/Notations3.out

```diff
@@ -130,14 +130,12 @@ return (1, 2, 3, 4)
 ! '{{x, y}}, x.y = 0
      : Prop
 exists x : nat,
-  nat ->
-  exists y : nat,
-    nat -> exists '{{u, t}}, forall z1 : nat, z1 = 0 /\ x.y = 0 /\ u.t = 0
+  nat -> exists y : nat, nat -> exists_mixed '{{u, t}}, x.y = 0 /\ u.t = 0
```

Comment:
Not sure why it changed this way, cc @herbelin

Reply:
Printing of recursive notations involving binders does not take general alpha-conversion into account. But the removal of the renaming of `z` into `z1` made by the PR in the above example makes that the names now match without having any longer to require a non-trivial alpha-conversion. So printing works by chance and the PR is not in the wrong.